### ISSUE TITLE
TraderBot: support multiple trading configurations at the same time

### ISSUE THREAD

**Issue by Konard** [linksplatform/Bot#194]
It will allow to trade on the same instrument, but on different broker accounts. This way it will allow to test different settings.
It will also allow to trade multiple instruments at the same time, on the same broker account or on different ones.

**Comment by shinev01** [comment]
Yes, we need it, it will be the best innovation ever, just a brain breaker. After that, the world will be turned upside down.
